server/models/favourites.js

```diff
@@ -5,18 +5,18 @@ const getFavoutesModel = (sequelize, Sequelize) => {
   const favouriteSchema = {
     id: {
       type: Sequelize.UUID,
-      defaultValue: Sequelize.UUIDV4,
       primaryKey: true,
       unique: true,
-      allowNull: false
-    },
-
-    userId: {
-      type: Sequelize.UUID,
-      allowNull: false
+      allowNull: false,
+      defaultValue: Sequelize.UUIDV4
     },
-
     articleId: {
+      type: Sequelize.UUID
+    },
+    articleSlug: {
+      type: Sequelize.TEXT
```

Comment:
This should be a `string` and cannot be null

Reply:
allowNull should be false, it is true by default